proto/regen/ecocredit/basket/v1/events.proto

```diff
@@ -6,7 +6,7 @@ import "regen/ecocredit/basket/v1/types.proto";
 
 option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket";
 
-// EventCreate is an event emitted when a basket is created.
+// DEPRECATED: EventCreate is an event emitted when a basket is created.
```

Comment:
This event is not deprecated and does not include any fields that should be deprecated.

```suggestion
// EventCreate is an event emitted when a basket is created.
```